frontend/src/constants/dbconstants.js

```diff
@@ -0,0 +1,7 @@
+export const COLLECTION_TRIPS = "trips";
```

Comment:
I like the way you named the constants so I am going to change mine to this format. However, since this is in the constants folder already, how do you like the naming of the file to just be `databse.js?

Reply:
sure